### PR TITLE
this is a test commit

### PR DESCRIPTION
**Pull request by JaehoonSong12**
dsafdafdsfdsafdsafds
f_dsf_d
safsd

continuation of #5

 
```
fsdf
dsfds
fsdaf
dsf
sdfdsfdsfds
```
fdsfdsfdsfsd
**fdsfdsfsd**fdsf
dsfdsfdsfsdf
dsfdsfdsfsdf

### CODE REVIEW

**Review by acdonge — APPROVED** [review]
it's gooood!

File: README.md [hunk 1]
```diff
@@ -88,12 +94,12 @@ For detailed documentation, including textbook references, project specification
 ## Table of Contents
 - [AU-CS Suite (AUC)](#au-cs-suite-auc)
   - [About](#about)
+  - [Project Management Platform](#project-management-platform)
```
Comment (selline0824) [inline README.md:97]: Nice try diddy~

**Review by selline0824 — CHANGES_REQUESTED** [review]
I dont like ur comment

**Review by kshn0109 — CHANGES_REQUESTED** [review]
ddd

**Review by acdonge — APPROVED** [review]

**Review by selline0824 — APPROVED** [review]

**Review by wish0210 — APPROVED** [review]

**Review by kshn0109 — APPROVED** [review]

**Pull request merged**